packages/backend/src/graphql/schema.graphql

```diff
@@ -491,6 +491,7 @@ type QuoteResponse {
 input CreateOutgoingPaymentInput {
   paymentPointerId: String!
   quoteId: String!
+  grant: String!
```

Comment:
Could `grantId` be made optional for incoming/outgoing payments instead of requiring grants in the admin api?

Reply:
If there is no grant, than there is no `clientId`. Hence, we can't filter by `clientId`.

Reply:
I'm imagining sub-resources created via the admin API not having a `grantId`/`clientId`.
Get and list via the admin API would always be the equivalent of read-all / list-all. We don't check auth / grant.
That should prevent clients doing read / read-all / list / list-all queries via Open Payments. Sub-resources without `grantId`/`clientId` are query-able for read-all / list-all.